.drone.yml

```diff
@@ -702,6 +702,28 @@ steps:
   - yarn install --immutable
   image: grafana/build-container:1.5.3
   name: initialize
+- commands:
+  - git clone "https://$${GITHUB_TOKEN}@github.com/grafana/grafana-enterprise.git"
```

Comment:
Depending on how big grafana-enterprise is this could take a decent while to get the whole history. we should probably add a `--depth=1` flag here

Reply:
True that, good point.